app/models/comment.rb

```diff
@@ -2,9 +2,9 @@
 
 class Comment < ApplicationRecord
   belongs_to :post
-  belongs_to :user
+  belongs_to :author
```

Comment:
- [ ] There is no model named `Author`, so we need to specify the model being referred to here else we would receive errors here. Based on the feedback from the previous reviewer kindly add the `class_name` attribute.

  ```
  belongs_to :author, class_name: 'User'
  ```